lib/command.js

```diff
@@ -43,6 +43,11 @@ function Command(name, args, options, callback) {
   this.replyEncoding = options && options.replyEncoding;
   this.errorStack = options && options.errorStack;
   this.args = args ? _.flatten(args) : [];
+  if (options && options.keyPrefix) {
```

Comment:
this is costly to access options + keyPrefix each time, imo it shoud be cached in a var. Also options should be always an object?